bahmni/src/main/java/org/avni_integration_service/bahmni/service/PatientService.java

```diff
@@ -119,7 +120,8 @@ private OpenMRSPatient createPatient(Subject subject, SubjectToPatientMetaData m
         String phoneNumberUuid = metaData.getPersonAttributesMappingList().getBahmniValueForAvniValue("Phone Number");
         if (subject.getObservation("Phone Number") != null && phoneNumberUuid != null){
             OpenMRSSavePersonAttributeType phoneNumberAttributeType = new OpenMRSSavePersonAttributeType(phoneNumberUuid, "phoneNumber");
-            OpenMRSSavePersonAttribute phoneNumberAttribute = new OpenMRSSavePersonAttribute(phoneNumberAttributeType, (String) subject.getObservation("Phone Number"));
+            Map<String, Object> phoneNumberObs =  (Map<String, Object>) subject.getObservation("Phone Number");
```

Comment:
@kushboojain-iplit Can you make this backward-compatible ? This will break for implementations running with older version of avni-serer